pkgs/by-name/mo/mosml/package.nix

```diff
@@ -4,6 +4,7 @@
   fetchFromGitHub,
   gmp,
   perl,
+  gcc13,
```

Comment:
you should switch from stdenv to gcc13Stdenv instead adding gcc13

Reply:
Very well. I did so by putting back a `mosml` field in `all-packages.nix`.